orchestration/dagster_orchestration/hca_manage/soft_delete.py

```diff
@@ -24,6 +35,92 @@ def run(arguments: Optional[list[str]] = None) -> None:
 
 
 def soft_delete(args: argparse.Namespace, host: str) -> None:
-    hca = HcaManage(environment=args.env, data_repo_client=get_api_client(host=host),
-                    project=args.project, dataset=args.dataset)
+    hca = SoftDeleteManager(environment=args.env, data_repo_client=get_api_client(host=host),
+                            project=args.project, dataset=args.dataset)
     hca.soft_delete_rows(args.path, args.target_table)
+
+
+@dataclass
+class SoftDeleteManager:
+    environment: str
+    data_repo_client: RepositoryApi
+    project: Optional[str] = None
+    dataset: Optional[str] = None
+
+    bucket_project: str = field(init=False)
+    bucket: str = field(init=False)
+
+    def __post_init__(self) -> None:
+        self.filename_template = f"sd-{self.project}-{self.dataset}-{uuid.uuid4()}-{{table}}.csv"
+        self.bucket_project = {"prod": "mystical-slate-284720",
```

Comment:
let's pull out hardcoded config maps like this into properties of the dataclass that will be passed in. i don't think we should have hardcoded config in this repo at all at this point, but if it's out of scope to remove this in this pr, let's keep it in a top-level constant.